### ISSUE TITLE
React Native requires Babel devDependencies

### ISSUE THREAD

**Issue by jonlong** [davidkpiano/react-redux-form#26]
Hey, thanks for this lib! I'm excited to start building with it — on first glance it looks like exactly what I'm after 😀.

I'm using React Native and noticed that I needed to install the following `devDependencies` in order to get `react-redux-form` to compile:

```
    "babel-core": "^6.3.17",
    "babel-preset-es2015": "^6.3.13",
    "babel-preset-react": "^6.3.13",
    "babel-preset-stage-0": "^6.3.13",
```

This is my first foray into RN, so I'm not sure how its Babel compiler works in cases like these. Should this be a necessary step for using `react-redux-form` with RN, or is there a better way I'm overlooking?

**Comment by davidkpiano** [comment]
Interesting, that might be a React Native issue rather than a React-Redux-Form issue, since RRF exposes `/lib/index.js`, which is already pretranspiled into ES5 code, so Babel isn't even necessary.

Let me know if you have any more issues with React Native and I'll respond as soon as I can!

**Comment by MiMaMuh** [comment]
@davidkpiano Got the same issue, don't know why. I removed the .babelrc from the installed module and it works now. It's not really a solution, but maybe you could exclude it with your npm-ignore from your package like this: https://github.com/kadirahq/npm-base/blob/master/.npmignore?

**Comment by davidkpiano** [comment]
Sure thing, @MiMaMuh. Doing that now.

**Comment by jonlong** [comment]
@MiMaMuh @davidkpiano thanks for revisiting this! I did a little more research, and the root of it seems to be RN's move to Babel 6. Here's a thread with more info:  https://github.com/facebook/react-native/issues/4062

The team's [official fix](https://github.com/facebook/react-native/issues/4062#issuecomment-179944792) is just as @MiMaMuh suggested.

**Comment by davidkpiano** [comment]
Should be available in `0.9.11` now! @jonlong @MiMaMuh
